src/crypto/crypto_common.cc

```diff
@@ -767,9 +767,9 @@ static bool PrintGeneralName(const BIOPointer& out, const GENERAL_NAME* gen) {
       return false;
     }
     char* oline = nullptr;
-    size_t n_bytes = BIO_get_mem_data(tmp.get(), &oline);
-    CHECK_IMPLIES(n_bytes != 0, oline != nullptr);
-    PrintAltName(out, oline, n_bytes, true, nullptr);
+    long n_bytes = BIO_get_mem_data(tmp.get(), &oline);  // NOLINT(runtime/int)
+    CHECK_IMPLIES(n_bytes > 0, oline != nullptr);
```

Comment:
This change should not weaken the existing check. Also, unrelated to the existing check, I'd prefer to explicitly check that the return value is non-negative, even though this restricts the string to 2 GiB on 32 bit systems.

```suggestion
    CHECK(n_bytes >= 0);
    CHECK_IMPLIES(n_bytes != 0, oline != nullptr);
```